examples/nfs-pd/testpd.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta1
+desiredState:
+  manifest:
+    containers:
+      - name: testpd
+        image: dockerfile/nginx
+        volumeMounts:
+            # name must match the volume name below
+            - name: nfs
+              mountPath: "/var/www/html/mount-test"
```

Comment:
Perhaps update the example to v1beta3?  I see all other examples being updated at this time.